packages/vitest/src/node/reporters/renderers/windowedRenderer.ts

```diff
@@ -175,6 +174,38 @@ export class WindowRenderer {
   private write(message: string, type: 'output' | 'error' = 'output') {
     (this.streams[type] as Writable['write'])(message)
   }
+
+  private addProcessExitListeners() {
+    const onExit = (signal?: string | number) => {
```

Comment:
Use exit code that's passed here by Node:

```suggestion
    const onExit = (signal?: string | number, code?: number) => {
```